Icon/IconGenerator.swift

```diff
@@ -191,8 +208,10 @@ extension View {
 
     /// Generates an image from the current View
     /// - Parameter size: The size of the image to generate
-    func generateImageData(size: CGSize) throws -> Data {
-        let wrapper = NSHostingView(rootView: self)
+    func generateImageData(size: CGSize, frame: Bool) throws -> Data {
```

Comment:
☝️ I get what you mean by `frame` here, but I'm afraid it would be confused with a frame type. What about `clippingToFrame` or maybe just `rounded`?

Also, could you please update the above documentation to include the new parameter? That would probably also resolve any ambiguity.

Reply:
Sure, and another thing: the icon gets slightly too big then. If I just add a fix scaling factor `* 0.8`?